src/test/resources/TestSuites/All_operations_test_suite.xml

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="All operation suite">
+
+    <test name="Addition test">
+        <classes>
+            <class name="hw1.CalculatorAdditionTest"></class>
+        </classes>
+    </test>
+
+    <test name="Subtraction test">
+        <classes>
+            <class name="hw1.CalculatorSubtractionTest"></class>
+        </classes>
+    </test>
+
+    <test name="Multiplication test">
+        <classes>
+            <class name="hw1.CalculatorMultiplicationTest"></class>
+        </classes>
+    </test>
+
+    <test name="Division test">
+        <classes>
+            <class name="hw1.CalculatorDivisionTest"></class>
+        </classes>
+    </test>
```

Comment:
Why do you decide include all classes to the suite?

Reply:
> Why do you decide include all classes to the suite?

Because there is a requirement for homework: "Create a test suite, which could run all tests". I can rewrite this suite(include suite in it, not classes), but it seems to be similar.